wavefront_sdk/common/application_tags.py

```diff
@@ -0,0 +1,63 @@
+"""
+Metadata about your application represented as tags in Wavefront.
+
+@author: Hao Song (songhao@vmware.com)
+"""
+
+from wavefront_sdk.common.constants import APPLICATION_TAG_KEY, \
+    SERVICE_TAG_KEY, CLUSTER_TAG_KEY, SHARD_TAG_KEY, NULL_TAG_VAL
+
+
+class ApplicationTags:
+    """Metadata about your application represented as tags in Wavefront."""
+
+    def __init__(self, application, service, cluster=None, shard=None,
+                 custom_tags=None):
+        """
+        Construct ApplicationTags.
+
+        @param application: Application Name
+        @param service: Service Name
+        @param cluster: Cluster Name
+        @param shard: Shard Name
+        @param custom_tags: List of pairs of custom tags
+        """
+        if not application:
```

Comment:
This is useless unless you want to check it's not `None`, `False`, `''`,  `[]`, 0, and so on

Reply:
I mean
```
>>> import wavefront_sdk
>>> wavefront_sdk.common.application_tags.ApplicationTags()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() missing 2 required positional arguments: 'application' and 'service'
```

Reply:
mainly want to make sure they're not passing `''` for `'application'` and `'service'`